### ISSUE TITLE
Improve outlines

### ISSUE THREAD

**Issue by dolanske** [dolanske/vui#38]
All focusable elements should have a 2px text-colored outline

**Comment by dolanske** [comment]
So far improved all input elements

**Comment by dolanske** [comment]
Everything else has been updated
